exercises/robot-name/.meta/solutions/RobotName.pm

```diff
@@ -1,21 +1,19 @@
+# This will be class 'RobotName', defined using Moo (https://perlmaven.com/oop-with-moo)
 package RobotName;
-use warnings;
-use strict;
+use Moo;
 
-sub new {
-  my $class = shift;
-  return bless {}, $class;
-}
+has name => ( is => 'rwp' ); # rwp = read-write protected: read-only to
+      # consumers, but settable using $self->_set_name
```

Comment:
Looks like perltidy wants to realign these comments. Perhaps it would be worth putting them both on their own lines in the template?